internal/builders/execaccessbuilder/create_access_resources.go

```diff
@@ -61,14 +61,11 @@ func (b *ExecAccessBuilder) CreateAccessResources(
 		return statusString, err
 	}
 
-	// Generate the user-friendly information for how to access the pod
-	//
-	// TODO: Templatize this into the ExecAccessTemplate in some way
-	//
 	accessString := fmt.Sprintf(
-		"kubectl exec -ti -n %s %s -- /bin/sh",
+		"kubectl exec -ti -n %s %s -- %s",
```

Comment:
I'd really like to turn this entire string into a template - not just the `/bin/sh` part. The reason is that an access request could tell the user to do something totally different like `kubectl cp ...` or perhaps a teams environment doesn't use `kubectl` directly but instead they use some internal tool to wrap it.

Ideally someone could set `accessString` to something like this: `kubectl exec -ti -n {{ .metadata.namespace }} {{ .metadata.name }} -- /bin/sh`. This would require running the string through a template tool, and being able to pass in the data about the request. There's a little thought to be put into whether we hard code a list of allowed template variables (reasonable), or we make it totally abstract and allow users to reference any value from the PodAccessRequest as an example. What do you think?

Reply:
I think it's more appropriate to set these variables into the PodAccessTemplate than the PodAccessRequest. 

I'm not sure how we'd handle something like the `targetPodName` as the `{{ .metadata.name }}` as this could not be hardcoded on either the request or template ?